scripts/state/collision/collision.js

```diff
@@ -0,0 +1,60 @@
+class Collision {
+
+  collisions(allBullets, allShips, sounds) {
+    let bulletsVis = [];
```

Comment:
makes it so I don't iterate over a ton of non visible bullets, lessens the amount of work alot